pokemongo_bot/__init__.py

```diff
@@ -39,12 +43,31 @@ def __init__(self, config):
         self.latest_inventory = None
         self.cell = None
 
+
     def start(self):
         self._setup_logging()
         self._setup_api()
         self.navigator = SpiralNavigator(self)
         random.seed()
 
+    def _setup_event_system(self):
```

Comment:
@TheSavior the event system is set up before the api setup (where bag info is being print). We can still use the event system to log it.